### DISCUSSION TITLE
Change what the event returns

### DISCUSSION THREAD

**Discussion by luzybry94** [JedWatson/react-select#4607] [Q&A]
Hi,

`const options = [
  { value: "chocolate", label: "Chocolate" },
  { value: "strawberry", label: "Strawberry" },
  { value: "vanilla", label: "Vanilla" },
];`

I was wondering if there was a way to configure React-Select so that the `event` triggered returns just the `value` selected instead of an array containing an object with the above key/value pairs?

Thanks

**Comment by ebonow** [comment]
Greetings @luzybry94 ,

There is an `actionMeta` object returned as the second parameter of onChange that might be what you are looking for. 

```js
const onChange = (value, actionMeta) => {
  const { action, option } = actionMeta;
  
  if (action === 'select-option') {
    console.log('This is likely what you are looking for: ', option);
  }
} 
```
